crates/ruff_python_codegen/src/stylist.rs

```diff
@@ -212,7 +212,7 @@ x = (
         let locator = Locator::new(contents);
         let program = parse_module(contents).unwrap();
         let stylist = Stylist::from_tokens(program.tokens(), &locator);
-        assert_eq!(stylist.indentation(), &Indentation::default());
+        assert_eq!(stylist.indentation(), &Indentation("  ".to_string()));
```

Comment:
What's the reason for changing the expected output in this test?

Reply:
This is coming from `main` via https://github.com/astral-sh/ruff/commit/bd46cd1fcf3dbc0da1cff763d061293c9bc09087 but I failed to recognize that while rebasing.